### PR TITLE
feat: cssinjs for image

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     image组件改为 cssinjs      |
| 🇨🇳 Chinese |    cssinjs for image       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/_util/theme/default.tsx [hunk 1]
```diff
@@ -78,6 +81,17 @@ const defaultDesignToken: DesignToken = {
   duration: 0.3,
 
   zIndexDropdown: 1050,
+  zIndexImage: 1080,
+  zIndexModalMask: 1000,
+
+  // Image
+  animationDurationSlow: '0.3s',
```
Comment (zombieJ) [inline components/_util/theme/default.tsx:88]: 这些全部写到 image 的 style 文件里，加一个 `FIXME` 不做全局透出

File: components/_util/theme/default.tsx [hunk 2]
```diff
@@ -78,6 +81,17 @@ const defaultDesignToken: DesignToken = {
...
+  imageSizeBase: '48px',
```
Comment (zombieJ) [inline components/_util/theme/default.tsx:89]: 另外，数字相关的可以不用 `px`

File: components/image/style/index.tsx [hunk 3]
```diff
@@ -1,2 +1,310 @@
...
+    imagePreviewOperationColor,
+    modalMaskBg,
+    iconPrefixCls,
+    imagePreviewOperationDisabledColor,
+    previewPrefixCls,
+  } = token;
+  return {
+    position: 'absolute',
+    top: '50%',
+    right: '10px',
```
Comment (zombieJ) [inline components/image/style/index.tsx:123]: 使用逻辑位置 `inset-block-end` 代替

File: components/image/style/index.tsx [hunk 4]
```diff
@@ -1,2 +1,310 @@
...
+    zIndex: 1,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '44px',
```
Comment (zombieJ) [inline components/image/style/index.tsx:128]: hardcode 提取出来，另外不要加单位

Comment (zombieJ) [reply components/image/style/index.tsx:128]: 其他的加单位的都看一下，都可以删掉

File: components/image/style/index.tsx (hunk 3 above)
Comment (zombieJ) [reply components/image/style/index.tsx:123]: 其他使用 left & right 的都使用逻辑位置替换掉，这样RTL 就可以少些很多代码： https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties

File: components/_util/theme/default.tsx (hunk 1 above)
Comment (zombieJ) [reply components/_util/theme/default.tsx:88]: 参考这个：

https://github.com/ant-design/ant-design/blob/ae6240d9ccebf4ccbf1de9cafa68d854e530a054/components/slider/style/index.tsx#L279

**Comment by yoyo837** [comment]
push -f

**Pull request closed without merging**